### ISSUE TITLE
Error on line 40 and 46 when using default variables

### ISSUE THREAD

**Issue by lyze237** [EdricChan03/action-build-deploy-ghpages#3]
Hello!

I've setup a basic github actions system with your action, however it throws some exceptions:

```yml
name: Deploy Jekyll Website
on: 
  push:
    branches:
    - master
jobs:
  build:
    runs-on: ubuntu-latest
    steps:
    - name: Source checkout
      uses: actions/checkout@master
    - name: Build and deploy jekyll website
      uses: EdricChan03/action-build-deploy-ghpages@v2.2.0
      with:
        github_token: ${{ secrets.GITHUB_TOKEN }}
        gh_pages_token: ${{ secrets.GH_PAGES_TOKEN }}
        OVERRIDE_GH_PAGES_BRANCH: ${{ true }}
```

```

Run EdricChan03/action-build-deploy-ghpages@v2.2.0
/usr/bin/docker run --name bb81dd60631ac2e14781a983cf095e48fef9_8b1c5f --label 04bb81 --workdir /github/workspace --rm -e INPUT_GITHUB_TOKEN -e INPUT_GH_PAGES_TOKEN -e INPUT_OVERRIDE_GH_PAGES_BRANCH -e INPUT_GH_PAGES_BRANCH -e INPUT_GH_PAGES_DIST_FOLDER -e INPUT_GH_PAGES_COMMIT_MESSAGE -e INPUT_REMOTE_REPO -e INPUT_COMMITTER_USERNAME -e INPUT_COMMITTER_EMAIL -e INPUT_GIT_FORCE -e INPUT_GH_PAGES_ADD_NO_JEKYLL -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/lyze237.github.io/lyze237.github.io":"/github/workspace" 04bb81:dd60631ac2e14781a983cf095e48fef9
/entrypoint.sh: line 33: INPUT_GH_PAGES_COMMIT_MESSAGE: Deploy commit  using EdricChan03action-build-deploy-ghpages in Deploy Jekyll Website: syntax error in expression (error token is "commit  using EdricChan03action-build-deploy-ghpages in Deploy Jekyll Website")
Installing gem bundle...
/entrypoint.sh: line 40: INPUT_REMOTE_REPO: https://x-access-token:***@github.com/lyze237/lyze237.github.io.git: syntax error in expression (error token is "://x-access-token:***@github.com/lyze237/lyze237.github.io.git")
/entrypoint.sh: line 46: INPUT_COMMITTER_EMAIL: lyze237@users.noreply.github.com: syntax error: invalid arithmetic operator (error token is "@users.noreply.github.com")
##[error]Docker run failed with exit code 1
```

I'm not really great with bash but I think you're missing a `-` after `:` in most lines. (I've forked the project and trying out things)

**Comment by EdricChan03** [comment]
Yup, you've spotted the problematic code that caused the issue. I'll fix this in a future release if I have the time for it.

**Comment by EdricChan03** [comment]
Sorry for the long wait, but I've finally fixed this issue with commit 7e9ba67.

Thanks for your patience and inconvenience!
